### ISSUE TITLE
browser.sessions.* isn't polyfilled

### ISSUE THREAD

**Issue by eejdoowad** [mozilla/webextension-polyfill#52]

**Comment by rotemdan** [comment]
Since I needed this functionality I've made a custom build with the following added to `api-metadata.json`:
```json
  "sessions": {
    "getDevices": {
      "minArgs": 0,
      "maxArgs": 1
    },
    "getRecentlyClosed": {
      "minArgs": 0,
      "maxArgs": 1
    },
    "restore": {
      "minArgs": 1,
      "maxArgs": 1
    }
  }
```
and it seems to work so far.

According to the [chrome documentation](https://developer.chrome.com/extensions/sessions) these three methods are the only ones supported in Chrome.

Apparently there's an `onChanged` event as well but I'm not sure if it requires a polyfill of sort.

Is this sufficient for a pull request? or the [additional methods (currently Firefox-only)](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/sessions) are needed as well?

**Comment by Rob--W** [comment]
This was fixed in 69e58761a5a4a50abe768dd06ccc9b43617770e2, and will be released in 0.3.0 (see #88).

**Comment by Rob--W** [comment]
Fixed in 0.3.0: https://github.com/mozilla/webextension-polyfill/releases/tag/0.3.0
